assets/js/app.js

```diff
@@ -0,0 +1,38 @@
+// A User should see a start screen
+$('.game').html(AppTemplates.start(heroes));
+
+// Choose a character &
+// they are 'exported' to the battle screen
+$('.start-name').on('click', function() {
+  var index = $(this).data('index');
+  var game = new Game(heroes[index]);
+  $('.game').html(AppTemplates.battle(game));
+
+  // When you click on the attack button
+  // the chosen character attacks
+  $('.game').on('click', '.attack', function(ev) {
+    game.runAttack();
+  });
+
+  game.on('change', function() {
+    if (game.gameWin()) {
+      $('.game').html(AppTemplates.win(game));
+    } else {
```

Comment:
Should this be `if (game.gameWin()) else if(game.gameLose()) else`?

Right now you will always get a battle template